lib/sandbox/execute.js

```diff
@@ -211,18 +212,23 @@ module.exports = function (bridge, glob) {
             timers.clearEvent(id, err, res);
         });
 
+        postmanConsole = new PostmanConsole(bridge, options.cursor, options.debug && glob.console, execution);
+
         // send control to the function that executes the context and prepares the scope
         executeContext(scope, code, execution,
             // if a console is sent, we use it. otherwise this also prevents erroneous referencing to any console
             // inside this closure.
-            (new PostmanConsole(bridge, options.cursor, options.debug && glob.console, execution)),
+            postmanConsole,
             timers,
             (
                 new PostmanAPI(execution, function (request, callback) {
                     var eventId = timers.setEvent(callback);
 
                     dispatchEvent(bridge, execution, executionRequestEventName, options.cursor, id, eventId, request);
                 }, /* onStopExecution = */ function () {
+                    // Dispatch event to display system message on console informing user that the request
+                    // execution was skipped from script
+                    dispatchSystemMessage(postmanConsole, 'skip_request', _.get(execution, 'legacy._itemName'));
```

Comment:
Why dispatch a system type console event? Why not a "execution_stopped" or such event? Every client can then interpret it and use it however they want to.

Also, "system" type log is not supported in Console interface either.

Reply:
This can also be a part of any existing events like `item` event with some indication that the execution was stopped from script.